virtcontainers/sandbox.go

```diff
@@ -1166,6 +1167,11 @@ func (s *Sandbox) CreateContainer(contConfig ContainerConfig) (VCContainer, erro
 	defer func() {
 		// Rollback if error happens.
 		if err != nil {
+			s.Logger().Warningf("Container %q could not be created, stopping it", contConfig.ID)
+			if err = c.stop(true); err != nil {
```

Comment:
This line is overwriting the (first) error that should be returned by the function.

Reply:
Ahh, whoops! I should have done `err := c.stop` instead, right?

Reply:
No, you should use a new err variable instead. As we want err to be returned to the caller.